src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static Scanner sc = new Scanner(System.in);
+    private static Task[] list = new Task[100];
+    private static int taskCounter = 0;
+    private static String HORIZONTAL_LINE = "____________________________________________________________";
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo + "\n"
+                + HORIZONTAL_LINE + "\n"
+                + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + HORIZONTAL_LINE);
+
+        while (sc.hasNext()) {
+            String input = sc.nextLine();
+
+            if (input.equals("bye")) {
+                break;
+            
+            } else if (input.equals("list")) {
+                System.out.println(HORIZONTAL_LINE);
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < list.length; i++) {
+                    if (list[i] != null) {
+                        System.out.println(String.valueOf(i + 1) + "." + list[i]);
+                    } else {
+                        System.out.println(HORIZONTAL_LINE);
+                        break;
+                    }
+                }
+
+            } else if (input.contains("unmark ")) {
```

Comment:
```suggestion
            } else if (input.startsWith("unmark ")) {
```
Could [startsWith](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#startsWith(java.lang.String)) be more appropriate?